doc/api/next_api_changes/behavior/18193-BKB.rst

```diff
@@ -0,0 +1,11 @@
+ID attribute of XML tags in SVG files now based on SHA512 rather than MD5
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The `RenderSVG._make_id` method generates unique ID attributes for various
```

Comment:
I don't think we need to reference private functions. You should rewrite this to talk about SVG file saving only, and not internals.

Reply:
OK. That sounds good to me.